common/ulsr_hil_client.py

```diff
@@ -309,8 +323,51 @@ def connect_node_to_project(hil_client, node, project):
         hil_client.project.connect(project, node)
         log_info('Node `%s` connected to project `%s`' % (node, project))
     except FailedAPICallException as e:
-        log_error('Unable to connect node `%s` to project `%s`' % (node, project))
+        log_error('Unable to connect node `%s` to project `%s`' %
+                  (node, project))
         raise HILClientFailure(e)
     except ConnectionError as e:
         log_error("connect_node_to_project: Couldn't connect to HIL server.")
         raise HILClientFailure(e)
+
+
+def connect_network(hil_client, node, nic, network, channel):
```

Comment:
should we use the underscore notation to mark that it's a helper?

Reply:
From the [docs](https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles) 
> _single_leading_underscore: weak "internal use" indicator. E.g. from M import * does not import objects whose name starts with an underscore.

This helper can actually be imported in some other module, so I don't really need to actively stop it from being imported.

Reply:
if you think that this can be used in some other module, wouldn't it be a good idea to move it to a file with such common definitions?
I agree with @Izhmash that if it is a private method add an '_' to it
or if you think it can be used by other modules, move all such functions to a common library file.